example/res/layout/upi_payment_activity.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/layout"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <EditText
```

Comment:
Wrap in `TextInputLayout` and change to `TextInputEditText` (see `res/layout/create_sepa_debit_activity.xml`)

Also add hint text, maybe `"9999999999@upi"`?